node/node.go

```diff
@@ -275,15 +297,14 @@ func NewNode(
 
 	var blobVersionParams *corev2.BlobVersionParameterMap
 	if config.EnableV2 {
-		ctx := context.Background()
 		// 12s per block
 		ttl := time.Duration(blockStaleMeasure+storeDurationBlocks) * 12 * time.Second
 		n.ValidatorStore, err = NewValidatorStore(logger, config, time.Now, ttl, reg)
 		if err != nil {
 			return nil, fmt.Errorf("failed to create new store v2: %w", err)
 		}
 
-		blobParams, err := tx.GetAllVersionedBlobParams(ctx)
+		blobParams, err := tx.GetAllVersionedBlobParams(context.Background())
```

Comment:
what's the reason for removing the `ctx` variable, and getting a new context for every call? I suspect merge conflicts

Reply:
I had a bunch of changes in this branch that I decided were not necessary, so I reverted them. This was an unintended side effect of my reversion. I've fixed this so that it is the same as before.